spec/models/manageiq/providers/ovirt/infra_manager/refresher_spec.rb

```diff
@@ -0,0 +1,734 @@
+require_relative 'refresher/ovirt_refresher_spec_common'
+
+describe ManageIQ::Providers::Ovirt::InfraManager::Refresher do
+  include Spec::Support::EmsRefreshHelper
+  include OvirtRefresherSpecCommon
+
+  describe "#refresh" do
+    let(:zone) { zone = EvmSpecHelper.create_guid_miq_server_zone.last }
+    let!(:ems) do
+      secrets = Rails.application.secrets.ovirt
+
+      FactoryBot.create(:ems_ovirt, :zone => zone, :hostname => secrets[:hostname], :ipaddress => secrets[:ipaddress], :port => 443).tap do |ems|
+        ems.update_authentication(:default => {:userid => secrets[:username], :password => secrets[:password]})
+      end
```

Comment:
We can use a much more common way of setting up an ems factory without `init_defaults`